src/DropdownMenu.js

```diff
@@ -16,11 +70,19 @@ const DropdownMenu = React.createClass({
         'dropdown-menu-right': this.props.pullRight
       };
 
+    // Remove outline on focus by default
+    let style = this.props.style || {};
+    style.outline = style.outline || 'none';
+
+    // Adding tabIndex -1 to allow for focus, but not tabable
     return (
         <ul
+          tabIndex="-1"
```

Comment:
Ah, no, that tabIndex allows the list itself to be focusable so that it can listen to keyDown events. The items in the list are focused when those keyDown events occur. The items do not need "tabIndex" as they are "a"-tags. Should be very careful messing around with tabIndexes too as those are global, only "-1" is safe as it does not interfere with tabbing, only makes the element focusable.

The original bootstrap listens tot he keyDown events on document and each dropdown instance checks if it is open and then reacts to that. But since the react-bootstrap dropdowns themselves do not know if they are open or not, only their parent does, I chose to focus the menu itself to listen for those keyDown events.

An alternative would be to pass the "open" state of the parent to the menu, but that is less flexible I think.